packages/firestore/test/integration/api/query.test.ts

```diff
@@ -1614,6 +1615,34 @@ apiDescribe('Queries', (persistence: boolean) => {
       });
     });
   });
+
+  // eslint-disable-next-line no-restricted-properties
+  (persistence ? it.skip : it)(
+    'can raise expected snapshot when resume query after deleting docs',
+    () => {
+      const testDocs = {};
+      for (let i = 1; i <= 100; i++) {
+        Object.assign(testDocs, { ['doc' + i]: { key: i } });
```

Comment:
Can

```ts
Object.assign(testDocs, { ['doc' + i]: { key: i } })
```

be simplified to

```ts
testDocs['doc' + i] = { key: i }
```

Reply:
Typescript is not quite happy about with that:
<img width="1366" alt="Screenshot 2023-02-16 at 6 26 04 PM" src="https://user-images.githubusercontent.com/107142260/219533953-4a64daf9-cdef-4927-ad13-77a7dc3e49d2.png">

Reply:
We can convince TypeScript to be happy. Does this work:

Change

```ts
const testDocs = {};
```

to

```ts
const testDocs: { [key: string]: Object } = {}; 
```